scripts/lib/get_tests.py

```diff
@@ -65,6 +65,7 @@
                              "ERR.f45_g37_rx1.A",
                              "ERP.f45_g37_rx1.A",
                              "SMS_D_Ln9.f19_g16_rx1.A",
```

Comment:
I think that you should just replace the existing one rather than add a new one.

Reply:
Done!